packages/react-instantsearch-hooks/src/components/__tests__/InstantSearchSSRProvider.test.tsx

```diff
@@ -437,112 +466,4 @@ describe('InstantSearchSSRProvider', () => {
       ]);
     });
   });
-
-  // Fixes https://github.com/algolia/react-instantsearch/issues/3530
-  test('renders initial refinement and allows to refine them', async () => {
```

Comment:
mentioned in a different comment too, but this test wasn't actually asserting what it was meant to (toBeChecked at the bottom should be .not.toBeChecked if we actually fixed the issue, but we never did). The similar assertion about it being interactive is added to the other test earlier.